### PR TITLE
Add coverage for repository disable functionality (BZ1344391)

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_repository.py -m test_positive_reposet_disable_after_manifest_deleted
.
----------------------------------------------------------------------
Ran 1 test in 180.629s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_repository.py [hunk 1]
```diff
@@ -1743,6 +1743,49 @@ def test_positive_reposet_disable(self):
...
+            session.nav.go_to_select_org(org.name)
+            # Enable RH repository
+            self.sync.enable_rh_repos(repos, REPO_TAB['rpms'])
+            session.nav.go_to_sync_status()
+            # Sync the repo and verify sync was successful
+            self.assertTrue(self.sync.sync_noversion_rh_repos(
+                PRDS['rhel'], [REPOS['rhst6']['name']]
+            ))
+            # Delete manifest
+            sub.delete_manifest(data={'organization_id': org.id})
```
Comment (abalakh) [inline tests/foreman/ui/test_repository.py]: Is that data really required? It's ugly if it is :) - you've already passed the organization and actual api path contains org id so it's basically duplication. Could you please check? If it's not required you could use `entities.Subscription(organization=org).delete_manifest()` directly.

Comment (oshtaier) [reply tests/foreman/ui/test_repository.py]:
```
======================================================================
ERROR: Enable RH repo and sync it. Remove manifest and then disable
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/oshtaier/Desktop/robottelo/tests/foreman/ui/test_repository.py", line 1775, in test_positive_reposet_disable_after_manifest_deleted
    sub.delete_manifest(organization=org)
  File "/home/oshtaier/Desktop/nailgun/nailgun/entities.py", line 5395, in delete_manifest
    self._org_path('delete_manifest', kwargs['data']),
KeyError: 'data'
```

Comment (abalakh) [reply tests/foreman/ui/test_repository.py]: It asks for empty dict here `.delete_manifest({})`, but nevermind, not worth spending time on that, was rather just curious.

File: tests/foreman/ui/test_repository.py [hunk 2]
```diff
@@ -1743,6 +1743,49 @@ def test_positive_reposet_disable(self):
...
+        :expectedresults: RH repo was disabled
+
+        :BZ: 1344391
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        sub = entities.Subscription(organization=org)
+        with manifests.clone() as manifest:
+            upload_manifest(org.id, manifest.content)
```
Comment (ldjebran) [inline tests/foreman/ui/test_repository.py]: we can use: 
```
self.upload_manifest(org.id, manifests.clone())
```

**Pull request merged**